integration_test/grpcclient_test.go

```diff
@@ -0,0 +1,148 @@
+package integrationtest
```

Comment:
Nit: what about removing `_test` from the file name? this can help to clarify whether files have test cases or not IMO

Reply:
Counter example:
These files will surely be used only for testing, but once we meausure the code coverage, these test util would also be counted as not tested if we rename it

Reply:
they are created for testing but are not test cases themselves. it's more like util, so I thought they also would be good to be counted if we want to measure the exact coverage.